fv3core/stencils/ray_fast.py

```diff
@@ -30,18 +28,21 @@ def dm_layer(rf, dp, wind):
 
 
 @gtstencil()
-def dm_stencil(
-    dp: sd,
-    dm: sd,
-    pfull: sd,
-    rf: sd,
+def ray_fast_wind(
+    u: FloatField,
+    v: FloatField,
+    w: FloatField,
+    dp: FloatField,
+    pfull: FloatField,
     dt: float,
     ptop: float,
     rf_cutoff_nudge: float,
     ks: int,
+    hydrostatic: bool,
 ):
-    from __externals__ import namelist
+    from __externals__ import local_ie, local_je, namelist
```

Comment:
Now that we have this feature, we can do this! Awesome. Previously we had some discussion whether it would make sense to keep 2 stencils, and then merge with the 2 stencils that have the equivalent bounds in nh_p_grad. But without a way to abstract whole stencils, that approach would force us to inline different model ideas into the same place, making it difficult to test and identify those areas. What do you think? I am ok with using the local indices approach,  at least for now, we can always reconsider making 2 stencils and merging with nh_p_grad. Once we merge more stencils that will be one we reconsider -- merge these stencils even though they are different ideas or keep them separate. Then whether it is 1 stencil with local end specifications or 2 stencils with different bounds... probably going with 1 stencil is the way to push for given the hit each stencil call brings.

Reply:
I did not realize this was merge-able with `nh_p_grad`, can we make that a separate PR?

Reply:
yes totally, that brings up issues of how to test, whether to drop the smaller tests or not etc. So should be a separate task. Also, first nh_p_grad needs to be squished into 1-2 stencils as well.